benchmarks/swe_bench/requirements.txt

```diff
@@ -0,0 +1 @@
+swebench
```

Comment:
Should we add `docker` here because we use it directly? (i.e. it's not merely used by `swebench` but also imported by our code)